src/types/types.ts

```diff
@@ -1588,7 +1588,7 @@ export interface EdgeUserInfo {
 
 export interface EdgeContextEvents {
   close: void
-  error: Error
+  error: any // Will change to `unknown`
```

Comment:
Why change it to `any`?

Reply:
We have no way to guarantee that the field will be an `Error` object. It could be an error or a string or `null` or whatever else gets passed to `throw` deep in some library. Since we can't make this promise, we should just be honest about that fact.